.cloud-build/notebook-index-cloudbuild.yaml

```diff
@@ -0,0 +1,33 @@
+steps:
+  # Show the gcloud info and check if gcloud exists
+  - name: ${_PYTHON_IMAGE}
+    entrypoint: /bin/sh
+    args:
+    - -c
+    - gcloud config list --quiet
+  # Check the Python version
+  - name: ${_PYTHON_IMAGE}
+    entrypoint: /bin/sh
+    args:
+    - -c
+    - python3 .cloud-build/CheckPythonVersion.py -q
+  # Fetch full repo for diff purposes
+  - name: gcr.io/cloud-builders/git
+    args: [fetch, --unshallow, --quiet]
+  # Create a virtual environment
+  - name: ${_PYTHON_IMAGE}
+    entrypoint: /bin/sh
+    args:
+    - -c
+    - python3 -m venv workspace/env
+  # Install Python dependencies and run index generation script
+  - name: ${_PYTHON_IMAGE}
+    entrypoint: /bin/sh
+    args:
+    - -c
+    - |
+      . workspace/env/bin/activate &&
+      python3 notebooks/notebook_template_review.py --web --title -steps --desc --linkback --notebook-dir=notebooks/official
```

Comment:
This just writes in the filesystem of the test runner container so you won't be able to access it. You have to store somewhere in the cloud to access it.

Consider: https://cloud.google.com/build/docs/building/store-artifacts-in-cloud-storage